WtWSMS/common/culture/names_equivalency/00_names.txt

```diff
@@ -336,18 +331,18 @@ anthemios_male = { Anthemios Anthemius }
 antheria_female = { Antheria }
 antherius_male = { Andaro Anterio Anteru }
 anthimos_male = { Anthimos Anthinos }
-anthony_male = { A_done_nu Andolin Annwn AntO_ni Anthony Antn_nino Anto_nio Antoine Antoinus Anton Anton_inui Anton_u Anton_ui Antoni Antoni_n Antonin Antonine Antonineh Antonino Antoninu Antoninus Antonio Antonios Antoniotto Antoniu Antonius Antonolo Antouni Antuiy Antun_inu Antun_iny Antune Antuninu Antuone }
+anthony_male = { Andolin Annwn AntO_ni Anthony Anto_nio Antoine Anton Anton_u Anton_ui Antoni Antonio Antonios Antoniotto Antoniu Antonius Antonolo Antouni Antuiy Antune Antuone }
```

Comment:
Why the removal of the equivalence between Antoninus and Anthony?

Reply:
> Why the removal of the equivalence between Antoninus and Anthony?

Because these are different names, even if the first one originated from the second. I created a separate equivalence for Antoninus and its forms.